ci/lava/tests/test-mbed-crypto.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
```

Comment:
I guess in this file few things need to be changed. There are references to systemd which clearly don't have anything to do with it.

Reply:
Agreed!